metricbeat/module/jolokia/jmx/data.go

```diff
@@ -90,7 +91,22 @@ type Entry struct {
 //        "timestamp": 1519409583
 //        "status": 200,
 //     }
-//  }
+//  ]
+//
+// A response with single value
+//
+//  [
+//     {
+//      "request":{
+//          "mbean":"java.lang:type=Runtime",
+//         "attribute":"Uptime",
+//         "type":"read"
+//       },
+//        "value":88622,
+//         "timestamp":1551739190,
+//         "status":200
+//    }
+// ]
```

Comment:
Not so important as it is a comment, but it seems that there is a missing `}`, and indentation is a bit broken.

Reply:
Good catch! I couldnt see where missed the `}` but i fixed all the indentation in next commit 👍

Reply:
Oh, there wasn't a missing `}` sorry, I think identation tricked me.